resources/views/calcs/result.blade.php

```diff
@@ -7,8 +7,7 @@
  </head>
 
  <body>
-     <h2>計算アプリ</h2>
-     <p>
+     <h2> 計算アプリ</h2>
```

Comment:
`計算アプリ`の前の半角スペースは意図したものでしょうか？？
もし余白の調整であれば、文字列では行わずにCSSで行うので、ここでの半角スペースは不要です。。

```suggestion
     <h2>計算アプリ</h2>
```

Reply:
色々試して消し忘れただけです。
修正致しました。